extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -138,12 +165,38 @@ const buildExpandedViewOverlay = element => htmlFor(element)`
       </span>
     </div>`;
 
+/**
+ * Updates embed's corresponding <style> element with embedData.
+ * @param {!Element} embedStyleEl
+ * @param {!Object} embedData
```

Comment:
It's not immediately clear to callers what this object should contain, so maybe we can make a typedef?

Reply:
Done.